shared/src/commonMain/kotlin/org/jetbrains/kotlinconf/navigation/KotlinConfNavHost.kt

```diff
@@ -138,31 +138,31 @@ fun NavGraphBuilder.screens(navController: NavHostController) {
             onPrivacyPolicy = { navController.navigate(PrivacyPolicyForVisitorsScreen) },
             onGeneralTerms = { navController.navigate(TermsOfUseScreen) },
             onWebsiteLink = { urlHandler.openUri(URLs.KOTLINCONF_HOMEPAGE) },
-            onBack = navController::popBackStack,
+            onBack = navController::navigateUp,
             onSpeaker = { speakerId -> navController.navigate(SpeakerDetailScreen(speakerId)) },
         )
     }
     composable<CodeOfConductScreen> {
-        CodeOfConduct(onBack = navController::popBackStack)
+        CodeOfConduct(onBack = navController::navigateUp)
```

Comment:
this change seems unrelated

Reply:
Yes, this part is a bit of a cheat, but I just noticed while going back-and-forth with theme settings a lot that I can pop everything off the backstack and end up with an empty frame, so I think it's time to make this change.

Reply:
an empty frame should be fixed already in the latest dev build

Reply:
`popBackStack` is the correct function. the `navigateUp` is required when you need to leave the app when the app was opened by a deeplink on the android, as I know